### PR TITLE
Fix SSL Proxy

### PR DESCRIPTION
**Pull request by Gee19**
Refs #SUP-45

### CODE REVIEW

File: sdetools/extlib/http_req.py [hunk 1]
```diff
@@ -209,7 +209,12 @@ def connect(self):
                                           ca_certs=self.ca_certs)
         if self.cert_reqs & ssl.CERT_REQUIRED:
             cert = self.sock.getpeercert()
-            hostname = self.host.split(':', 0)[0]
+
+            if self._tunnel_host:
```
Comment (funkaoshi) [inline sdetools/extlib/http_req.py]: Probably nicer to write this as:

```
host = self.host if not self._tunnel_host else self._tunnel_host
hostname = host(':', 0)[0]
```

Comment (Gee19) [inline sdetools/extlib/http_req.py]: Fixed, thanks ram

**Comment by funkaoshi** [comment]
+1

I like Shawn's suggestion in the other MR as well. Can maybe discuss that change more in the develop MR. We should merge this and make a new sdetools they can test with.

**Comment by funkaoshi** [comment]
@Gee19 Oh, you'll need to update the sdetools version here as well. grep for VERSION and you should find where.

**Comment by funkaoshi** [comment]
+1 we can make a new build.

**Pull request merged**